canonical-web-and-design/ubuntu.com
# Update notifications structure to new pattern - part 1

## Done

- The notification pattern was updated, and support for the old version of the pattern is removed in Vanilla 3.x, this PR updates many instances across the project to the new structure - see the upgrade guide: https://vanillaframework.io/docs/migration-guide-to-v3#notifications
- There are notifications on 100+ pages/templates, so there'll be at least one more follow up PR to make it slightly easier to review

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check the files that were changed, and find the pages on the demo, see that the notifications on them are not broken

## Issue / Card

(partially) fixes https://github.com/canonical-web-and-design/web-squad/issues/4854
